test/initialism/proto/test/initialism/v1/initialism.proto

```diff
@@ -0,0 +1,36 @@
+syntax = "proto3";
+
+// buf:lint:ignore PACKAGE_DIRECTORY_MATCH
+package mycompany.initialism;
+
+import "temporal/v1/temporal.proto";
+
+service AWS {
+  option (temporal.v1.service) = {task_queue: 'aws-task-queue'};
+
+  // ManageAWS does some workflow thing.
+  rpc ManageAWS(ManageAWSRequest) returns (ManageAWSResponse) {
+    option (temporal.v1.workflow) = {id: 'some-workflow-1/${! id }/${! uuid_v4() }'};
```

Comment:
```suggestion
    option (temporal.v1.workflow) = {id: 'some-workflow-1/${! urn }/${! uuid_v4() }'};
```

Reply:
Applied in another change where I fixed the `some-workflow-1` portion as well.